Setup/Patch/Data/CreateCategoryAttribute.php

```diff
@@ -59,7 +60,7 @@ public function apply()
      */
     public static function getDependencies()
     {
-        return [];
+        return [CreateCategoryAttribute::class];
```

Comment:
Why would `CreateCategoryAttribute` will be on itself?
Shouldn't `UpdateCategoryAttribute` be dependent on `CreateCategoryAttribute`?

Reply:
This was a mistake. Will fix